bundles/mapping/mapmodule/service/map.layer.js

```diff
@@ -525,6 +529,7 @@ Oskari.clazz.define('Oskari.mapframework.service.MapLayerService',
                 this.getSandbox().notifyAll(Oskari.eventBuilder('MapLayerEvent')(null, 'add'));
 
                 if (typeof callbackSuccess === 'function') {
+                    me.trigger("Layers.Loaded", true);
```

Comment:
Why? It already sends out a MapLayerEvent when this happens. In any case the event shouldn't be tied on having the callback function sent or not.